setup_lacework_agent.sh

```diff
@@ -2,24 +2,38 @@
 
 set -e
 
-LACEWORK_INSTALL_PATH="{{ LaceworkInstallPath }}"
```

Comment:
@JPLachance do you know if there is a reason why this variable should use double-quotes `""` instead of single-quotes `''`?

Reply:
There is one yes.

If I remember correctly, it's because of the way System Manager passes values to the script. For example, TAGS is a JSON object and using single quotes was breaking the JSON. To be consistent, I used double quotes everywhere for parameters passed by System Manager.

Make sense?

Feel free to leave a comment 😄